spikeextractors/extraction_tools.py

```diff
@@ -886,7 +886,7 @@ def cast_start_end_frame(start_frame, end_frame):
         end_frame = end_frame
     else:
         raise ValueError("end_frame must be an int, float (not infinity), or None")
-    return start_frame, end_frame
+    return start_frame.astype('int64'), end_frame.astype('int64')
```

Comment:
safety measure (see https://github.com/SpikeInterface/spiketoolkit/issues/409)

Reply:
I don't known underlying reason.
I would prefer a int(start_frame).
Force a cast to a fixed precision is not the spirit I think.